LibreNMS/Device/YamlDiscovery.php

```diff
@@ -188,6 +188,11 @@ public static function replaceValues($name, $index, $count, $def, $pre_cache)
             $variables = [
                 'index' => $index,
                 'count' => $count,
+                // we compute a numOid compatible version of index
+                // string length followed by ASCII of each char.
+                'index_string' => implode('.', array_map(function ($index) {
```

Comment:
This correct implementation is the fix for this old issue:  https://community.librenms.org/t/juniper-loss-not-polling-correctly/16628/3